app/src/components/ResourceInfoCard.tsx

```diff
@@ -1,8 +1,22 @@
-import { ActionIcon, Grid, Paper, Text, createStyles, em, getBreakpointValue, rem, Tooltip } from '@mantine/core';
+import {
+  ActionIcon,
+  Grid,
+  Paper,
+  Text,
+  createStyles,
+  em,
+  getBreakpointValue,
+  rem,
+  Tooltip,
+  Group
```

Comment:
I know you mentioned in the past we should add imports in alphabetical order. This might not apply since it is a different file but just thought I should mention it.